src/lib/views/AlbumsView.svelte

```diff
@@ -98,6 +100,12 @@
         isVisible = true;
         isInit = false;
     }
+    
+    $: if (container && $currentSong?.album.toLowerCase() !== currentAlbum?.title.toLowerCase()) {
```

Comment:
The `container` condition is added to make sure that it's existing so that `onScroll()` don't fail.